.github/workflows/ci.yml

```diff
@@ -76,8 +78,8 @@ jobs:
       - name: Install python deps
         run: python -m pip install psycopg2-binary xmltodict
       - name: Run smoketests
-        # Note: clear_database and replication only work in private
-        run: python -m smoketests ${{ matrix.smoketest_args }} -x clear_database replication
+        run: cargo ci smoketests -- ${{ matrix.smoketest_args }}
```

Comment:
little nit: I kinda preferred when the skipped tests were provided here, since they're going to be skipped on every platform. Not super important, but also not super important to change (although we could consider renaming `smoketests_args` to `extra_smoketest_args` to be clearer).

If you prefer it this way, that's fine with me.